<!DOCTYPE html>
<html lang="en">


  <head profile="https://twhiscock.github.io/">
    <title>Hiscock Lab</title>

  <link rel="icon" type="image/png" href="./img/logo2.png">

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <meta name="description" content="">
  <meta name="author" content="">

  <style>
  img {
    max-width: 100%;
    height: auto;
  }
  </style>


  <title>Hiscock Lab</title>

  <!-- Bootstrap core CSS -->
  <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

  <!-- Custom styles for this template -->
  <link href="css/scrolling-nav.css" rel="stylesheet">

</head>

<body id="page-top">

  <!-- Navigation -->
  <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top" id="mainNav">
    <div class="container">
      <a class="navbar-brand js-scroll-trigger" href="#page-top">Hiscock Lab</a>
      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <div class="collapse navbar-collapse" id="navbarResponsive">
        <ul class="navbar-nav ml-auto">
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="#home">Home</a>
          </li>
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="#research">Research</a>
          </li>
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="#members">Members</a>
          </li>
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="#pubs">Publications</a>
          </li>
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="#contact">Join us</a>
          </li>
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="#metal">Metal detecting</a>
          </li>
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="#about">About</a>
          </li>
        </ul>
      </div>
    </div>
  </nav>

  <section id="home">
    <div class="container">
      <div class="row">
        <div class="col-lg-8 mx-auto">
          <h2>Self-organization in development: <br>
            how do animals build themselves?</h2>
          <p class="lead"> We are a team of physicists, mathematicians and (theoretical) developmental biologists. We develop new theory, mathematical models and computational tools, in close collaborations with experimentalists, to understand development - the remarkable processes that allow single-cell embryos to self-organize into complex, functional and diverse adult forms. We are based at the Institute for Medical Sciences, University of Aberdeen.</p>
          <img src="./img/zebrabow2.png" id="photo" width="1000">
        </div>
      </div>
    </div>
  </section>

  <section id="research">
    <div class="container">
      <div class="row">
        <div class="col-lg-8 mx-auto">
          <h2>Research</h2>
          <p> Research in the group aims to understand how spatial organisation (patterns, shapes, forms) self-organize in developing embryos; a complex, inherently spatiotemporal, and often counterintuitive  phenomenon. Our guiding motivation is that, whilst decades of developmental biology has uncovered many essential molecular and cellular players, how these co-ordinately self-assemble into patterned organs and tissues remains much less well understood. Our approach is to develop mathematical models of developmental processes (often PDEs) to bridge this gap. We hope not only to understand something new about how embryos develop, but also more generally about the spatiotemporal dynamics of complex self-organising systems. We adapt existing theoretical frameworks where applicable, and formulate our own when the biology demands it. We are also developing new computational tools and software to accelerate and improve our models. 
          </p>
          <p> As theorists, we are privileged to be able to work on a wide range of problems and with a wide range of people. Some of the projects in the lab are primarily theoretical in nature, whereas others focus on one biological system in detail in collaboration with experimentalists. Throughout all these projects, we take a distinctively <i> theory-led approach </i>.  Here, theory takes centre stage, not just a tool to validate already well-understood mechanisms, but also to challenge our assumptions and sometimes even provoke new hypotheses, which are then later experimentally interrogated. To find up-to-date information on our current projects and interests, please check out our papers or get in touch directly.
          </p>

          <h4>Recent/current areas of focus</h4>
          <p> <b> Orientation mechanisms for self-organized stripes: </b> Unlike most patterns we see in nature, in silico Turing patterns are often disorganized and have variable orientation. <a href="https://www.cell.com/cell-systems/pdfExtended/S2405-4712(15)00215-X">We used theory</a> to predict rather generally how self-organizing stripes could be oriented in vivo, leading to three hypotheses for stripe orientation. <a href="https://www.biorxiv.org/content/10.1101/2023.09.25.559425v1.full.pdf">In subsequent work</a>, we teamed up with an experimentalist (Cliff Tabin/Evan Kingsley, Harvard University) to study the orientation of cartilage rings in the developing airways. We have discovered an unexpected role for a dynamic morphogen gradient, providing a mechanistic basis for one of our originally theory-led hypotheses. <br>
            <img src="./img/orientation.png" id="photo" width="800">
          </p>
          
          <p> <b> Generic properties of Turing-like periodic patterns: </b> Simple Turing models are commonly used to explain periodic patterns in development, however the relevance of Turing patterns in vivo remains controversial. <a href="https://dev.biologists.org/content/142/3/409.full-text.pdf?with-ds=yes">We recently proposed</a> a more general mathematical model for periodic patterning, which suggests that many different mechanisms (molecular, cellular, and mechanical) can generate qualitatively similar patterns to Turing systems. We are now investigating reaction-diffusion circuits in more detail, bringing the complexity of the models much closer to the complexity of the biological mechanisms. We are deriving general limits/bounds on pattern formation in these sophisticated reaction-diffusion circuits. 
          </p>

          <p> <b> Periodic patterning in the developing limb : </b> The developing limb is a paradigmatic example of pattern formation with decades worth of published data available. We have used mathematical modelling to re-interpret some of this data focussing on periodic patterning of the repeated joints of the fingers/toes. <a href="https://dev.biologists.org/content/147/8/dev183699.full-text.pdf?with-ds=yes">We first described a novel class of reaction-diffusion system</a> that could explain the repeated joint pattern. Together with Patrick Tschopp (University of Basel), <a href="https://doi.org/10.1073/pnas.2304470121">we have since identified</a> the molecular mechanism underpinning these dynamics, characterising, in our opinion, one of the first bona fide Turing mechanisms operating in vivo. <br>
            <img src="./img/joints.png" id="photo" width="1000">
          </p>
          <p> <b> Beyond periodic patterns: </b> Most of the self-organizing phenomena that we have focussed on so far concerns the generation periodic patterns i.e., those that repeat with a characteristic lengthscale. We have recently been awarded an <b>ERC Starting Grant </b> to explore novel self-organizing mechanisms - including the many developmental patterns that do not repeat - with a particular focus on reaction-diffusion mechanisms. 
          </p>




        </div>
      </div>
    </div>
  </section>

  <section id="members">
    <div class="container">
      <div class="row">
        <div class="col-lg-8 mx-auto">
          <h2>Lab members</h2>
          <table style="width:100%">
          <tr>
            <td  style="width:30%"> <img src="./img/tom.png" id="tom" width="150"> </td> <td> <b> Tom Hiscock, Group leader: </b> Tom trained as a theoretical physicist but fell in love with developmental biology during his PhD. He loves reading physics and biology papers in equal measure, and is fascinated by the remarkable ability of embryos to self-organize. Outside of work, he likes to move - hiking in the Cairngorms, cycling in Aberdeenshire, or swimming in a thick wetsuit! </td>
          </tr>
          <tr>
            <td  style="width:30%"> <img src="./img/Muzatko_small.jpg" id="tom" width="150"> </td> <td> <b> Daniel Muzatko, PhD student: </b> After completing a degree in Pure Mathematics, Daniel decided to apply what he learnt to real-life biological problems. His PhD project is focused on analysing and improving mathematical models of pattern formation. In his free time, he enjoys playing guitar, walking along cliffs, and playing board games with friends. </td>
          </tr>
          <tr>
            <td  style="width:30%"> <img src="./img/bijoy_small.jpg" id="tom" width="150"> </td> <td> <b> Bijoy Daga, postdoc: </b> Bijoy earned his PhD in physics and is now applying statistical mechanics and soft matter physics to unlock the secrets of developing embryos. He is currently collaborating with Ben Steventon's lab in Cambridge to understand the role of collective cell behavior in morphogenesis. Outside of the lab, Bijoy enjoys playing board games, exploring the Scottish countryside, visiting museums and binge-watching Netflix.</td>
          </tr>
          </table>


          <h3>Aberdeen Developmental Biology Group </h3>
          <img src="./img/lab2.jpg" id="tom" width="1000"> <p> Aberdeen Developmental Biology Group </p>


          <h2>Former lab members</h2>
          <p class="lead"> Students/postdocs </p>
          <ul style="list-style: none;">
            <li> <a href="https://www.crick.ac.uk/research/find-a-researcher/jake-cornwall-scoones"> Jake Cornwall Scoones </a> (now at the Crick) </li>
          </ul>
          <p class="lead">Undergrad/masters thesis project students </p>
          <ul style="list-style: none;">
            <li> Santa Walker, Shajjad Hossain, Molly Rennie, Sean Logue, Emily Sachde, Noor Chaudhary, Daniel Rennie  </li>
          </ul>
        </div>
      </div>
    </div>
  </section>

  <section id="pubs" class="bg-light">
    <div class="container">
      <div class="row">
        <div class="col-lg-12 mx-auto">
          <h2>Publications</h2>
          <table style="width:100%">
            <tr>
              <th> Year </th>
              <th> Title </th>
              <th> Authors </th>
              <th> Journal </th>
              <th> Links </th>
            </tr>
            <tr>
              <td> 2024 </td> <td> A dynamic Hedgehog gradient orients tracheal cartilage rings. </td> <td> Kingsley EP, Mishkind D, Hiscock TW&#8224, Tabin CJ&#8224</td> <td> biorXiv </td> <td> <a href="https://www.biorxiv.org/content/10.1101/2023.09.25.559425v1.full.pdf">pdf</a> </td>
            </tr>
            <tr>
              <td> 2024 </td> <td> Self-organized BMP signaling dynamics underlie the development and evolution of digit segmentation patterns in birds and mammals. </td> <td> Grall E, Feregrino C, Fischer S, De Courten A, Sacher F, Hiscock TW&#8224, Tschopp P&#8224 </td> <td> PNAS </td><td> <a href="https://doi.org/10.1073/pnas.2304470121">pdf</a> </td>
            </tr>
            <tr>
              <td> 2021 </td> <td> Secreted inhibitors drive the loss of regeneration competence in Xenopus limbs. </td> <td> Aztekin C*, Hiscock TW*, Gurdon J, Jullien J, Marioni J, Simons, BD </td> <td> Development </td><td> <a href="https://journals.biologists.com/Toolbox/DownloadCombinedArticleAndSupplmentPdf?resourceId=269060&multimediaId=2079165&pdfUrl=%2Fcob%2Fcontent_public%2Fjournal%2Fdev%2F148%2F11%2F10.1242_dev.199158%2F4%2Fdev199158.pdf">pdf</a> </td>
            </tr>   
            <tr>
              <td> 2020 </td> <td> A dot-stripe model of joint patterning in the tetrapod limb. </td> <td> Cornwall Scoones J, Hiscock TW&#8224 </td> <td> Development </td><td> <a href="https://dev.biologists.org/content/147/8/dev183699.full-text.pdf?with-ds=yes">pdf</a> </td>
            </tr> 
            <tr>
              <td> 2020 </td> <td> The myeloid lineage is required for the emergence of a regeneration-permissive environment following Xenopus tail amputation. </td> <td> Aztekin C, Hiscock TW, Butler R, Andino FDJ, Robert J, Gurdon JB, Jullien J.  </td><td> Development </td> <td> <a href="https://dev.biologists.org/content/develop/147/3/dev185496.full.pdf">pdf</a> </td>
            </tr> 
            <tr>
              <td> 2019 </td> <td> Adapting machine-learning algorithms to design gene circuits. </td> <td> Hiscock TW&#8224  </td> <td> BMC bioinformatics. </td><td> <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2788-3">pdf</a> </td>
            </tr> 
            <tr>
              <td> 2019 </td> <td> Navigating at night: fundamental limits on the sensitivity of radical pair magnetoreception under dim light. </td> <td> Hiscock HG, Hiscock TW, Kattnig DR, Scrivener T, Lewis AM, Manolopoulos DE, Hore PJ. </td><td> Quarterly reviews of biophysics </td> <td> <a href="https://ora.ox.ac.uk/objects/uuid:6dbe6bfe-d635-4bed-b5d0-896437f8044a/download_file?safe_filename=218%2520QRB.pdf&file_format=application%2Fpdf&type_of_work=Journal+article">pdf</a> </td>
            </tr> 
            <tr>
              <td> 2019 </td> <td> Identification of a regeneration-organizing cell in the Xenopus tail. </td> <td> Aztekin C*, Hiscock TW*, Marioni JC, Gurdon JB, Simons BD, Jullien J. </td><td> Science </td> <td> <a href="https://www.repository.cam.ac.uk/bitstream/handle/1810/292984/aav9996_CombinedPDF_v3.pdf?sequence=1&isAllowed=y">pdf</a> </td>
            </tr> 
            <tr>
              <td> 2019 </td> <td> A single-cell molecular map of mouse gastrulation and early organogenesis.  </td> <td> Pijuan-Sala B, Griffiths JA, Guibentif C, Hiscock TW, Jawaid W, Calero-Nieto FJ, et al. </td> <td> Nature </td><td> <a href="https://www.nature.com/articles/s41586-019-0933-9?platform=oscar&draft=collection">link</a> </td>
            </tr> 
            <tr>
              <td> 2018 </td> <td> Feedback between tissue packing and neurogenesis in the zebrafish neural tube. </td> <td> Hiscock TW, Miesfeld JB, Mosaliganti KR, Link BA, Megason SG. </td><td> Development </td> <td> <a href="https://dev.biologists.org/content/145/9/dev157040.full-text.pdf?with-ds=yes">pdf</a> </td>
            </tr> 
            <tr>
              <td> 2018 </td> <td> Size-reduced embryos reveal a gradient scaling-based mechanism for zebrafish somite formation. </td> <td> Ishimatsu K, Hiscock TW, Collins ZM, Sari DWK, Lischer K, Richmond DL, et al. </td><td> Development </td> <td> <a href="https://dev.biologists.org/content/145/11/dev161257.full-text.pdf?with-ds=yes">pdf</a> </td>
            </tr> 
            <tr>
              <td> 2018 </td> <td> Identity and novelty in the avian syrinx. </td> <td> Kingsley EP, Eliason CM, Riede T, Li Z, Hiscock TW, Farnsworth M, et al. </td><td> PNAS </td> <td> <a href="https://www.pnas.org/content/pnas/115/41/10209.full.pdf">pdf</a> </td>
            </tr> 
            <tr>
              <td> 2018 </td> <td> Observing the cell in its native state: Imaging subcellular dynamics in multicellular organisms. </td> <td> Liu T, Upadhyayula S et al. </td> <td> Science </td><td> <a href="https://science.sciencemag.org/content/360/6386/eaaq1392.full.pdf">pdf</a> </td>
            </tr> 
            <tr>
              <td> 2017 </td> <td> On the Formation of Digits and Joints during Limb Development. </td> <td> Hiscock TW, Tschopp P, Tabin CJ.</td> <td> Dev Cell </td><td> <a href="https://www.cell.com/action/showPdf?pii=S1534-5807%2817%2930347-7">pdf</a> </td>
            </tr> 
            <tr>
              <td> 2015 </td> <td> Orientation of Turing-like Patterns by Morphogen Gradients and Tissue Anisotropies. </td> <td> Hiscock TW&#8224, Megason SG&#8224 </td><td> Cell Systems </td> <td> <a href="https://www.cell.com/cell-systems/pdfExtended/S2405-4712(15)00215-X">pdf</a> </td>
            </tr> 
            <tr>
              <td> 2015 </td> <td> Mathematically guided approaches to distinguish models of periodic patterning. </td> <td> Hiscock TW, Megason SG </td> <td> Development </td><td> <a href="https://dev.biologists.org/content/142/3/409.full-text.pdf?with-ds=yes">pdf</a> </td>
            </tr> 
            <tr>
              <td> 2014 </td><td> Interplay of cell shape and division orientation promotes robust morphogenesis of developing epithelia. </td> <td> Xiong F, Ma W, Hiscock TW, Mosaliganti KR, Tentner AR, Brakke KA, et al. </td>  <td> Cell </td><td> <a href="https://www.cell.com/action/showPdf?pii=S0092-8674%2814%2901155-6">pdf</a> </td>
            </tr> 

          </table>
        </div>
      </div>
    </div>
  </section>


  <section id="contact">
    <div class="container">
      <div class="row">
        <div class="col-lg-8 mx-auto">
          <h2>Join us</h2>
          <p> We are always looking to recruit new people to join our talented and diverse team. Please get in touch via my email below to talk about opportunities. </p>
          <p class="lead">PhD students</p>
          <p> There are a number of upcoming opportunities for PhD positions (e.g., ERC-funded studentships (<a href="https://www.findaphd.com/phds/project/adapting-machine-learning-algorithms-to-learn-the-rules-of-embryonic-development/?p175103"> here </a> and <a href="https://www.findaphd.com/phds/project/mathematical-theories-of-symmetry-breaking-in-embryos-organs-and-organoids/?p175101"> here </a>, deadline 10 November), <a href="http://www.eastscotbiodtp.ac.uk/"> EASTBIO studentships </a>). Please email me if you are interested in understanding the wonders of embryo development using computational methods! I'd be happy to discuss if this is a good fit. </p>
          <p class="lead">Postdocs</p>
          <p> I am currently hiring a number of postdoctoral fellows as part of an exciting <b>ERC-funded project </b> looking at new mechanisms of reaction-diffusion based patterning. Please get in touch if you are interested. </p>
          <p class="lead">Independent fellowships</p>
          <p> I would be happy to support applications for independent postdoctoral fellowships on a range of topics, and can provide in-depth mentorship on project ideation and funding acquisition. If your research vision could benefit from my input, then please get in touch and we can discuss next steps. </p>
          <p>
          Some possible opportunities include: <a href="https://www.embo.org/funding-awards/fellowships/postdoctoral-fellowships"> EMBO Long Term Fellowships</a>, <a href="https://royalsociety.org/grants-schemes-awards/grants/newton-international/"> Newton International Fellowships</a>, <a href = "https://wellcome.org/grant-funding/schemes/early-career-awards" > Wellcome Early Career Awards </a>, <a href="https://www.royalcommission1851.org/awards/">Royal Commission for the Exhibition of 1851</a>, <a href="http://ec.europa.eu/research/mariecurieactions/">Marie Sk≈Çodowska-Curie Fellowships</a>, <a href="http://www.hfsp.org/funding/postdoctoral-fellowships">Human Frontier Science Program</a>, <a href="https://www.leverhulme.ac.uk/funding/grant-schemes/early-career-fellowships">Leverhulme Trust Early Career Fellowships</a>.
          </p>
          <p class="lead">Undergraduate students</p>
          <p> If you are an undergraduate looking for an interesting summer research project, please get in touch! BSDB offers <a href="http://bsdb.org/awards/gurdon-studentships-for-summer-vacation-work/">summer studentships</a>. </p>
        </div>
      </div>
    </div>
  </section>


  <section id="metal">
    <div class="container">
      <div class="row">
        <div class="col-lg-8 mx-auto">
          <h2>Metal detecting (aka lab culture)</h2>
          <p> I have never tried metal detecting*. But I think it is such a wonderful metaphor for being a researcher. Like metal detectorists, we are searching for treasure. Sometimes we'll have a good idea of where we should look; othertimes we will happen upon something valuable when we least expect it. Most of the time, we will find something that has been seen many times before (e.g., a crumpled coke can). But spend enough time searching, and we may find something truly surprising (e.g., a Roman mosaic). It's this unpredictability that gives research this addictive and thrilling sense of adventure. But it's also this unpredictability that challenges us, personally and professionally. We cannot guarantee, no matter how hard we try, that we'll find something interesting - or, more specifically, something that everyone else finds interesting! So, like a happy band of metal detectorists scanning a meadow on a Summer's evening, all we can do is focus on the tasks at hand - searching systematically, exploring new fields, turning over un-turned stones. And, along the way, building friendships and supportive communities with people from all walks of life who are also walking this exciting but frustrating path of discovery. In my group, we try to celebrate not just the rare occasions when we find treasure, but all the good time and effort we have spent looking for it and supporting each other while we do so.
          </p>
            <p>
            *This has been inspired by a recent BBC comedy "The Detectorists", with this rather pleasant <a href="https://www.youtube.com/watch?v=VKI44y_Hvlw">song</a> by Johnny Flynn.
          </p>



          </p>
        </div>
      </div>
    </div>
  </section>

  <section id="about">
    <div class="container">
      <div class="row">
        <div class="col-lg-8 mx-auto">
          <h2>About me</h2>
          <p>I originally trained as a physicist at the University of Cambridge, before setting off across the Atlantic to do my PhD at Harvard Medical School. In Sean Megason's lab, I fell in love with embryos, and spent the best part of six years watching them develop under a microscope. In my postdoc, supervised by John Marioni and Ben Simons at the University of Cambridge, I worked on several projects combining my training in physics with my love for developmental biology. In Summer 2020, I started as a Lecturer in Systems Biology at the <a href="https://www.abdn.ac.uk/ims/">Institute for Medical Sciences</a>, University of Aberdeen. My group is embedded within the strong developmental biology hub at the IMS, and benefits from proximity to the wonderful wildlife, mountains and scenery that Northeast Scotland has to offer.  </p>
          <h2>Contact</h2>
          <p> You can contact me at:   thomas.hiscock@abdn.ac.uk </p>

          <p><img src="./img/riverDee.png" id="dee" width="500"> <img src="./img/balmoralWoods.png" id="woods" width="500"></p>
          <p><i>Images of the beautiful autumn leaves just outside Aberdeen, October 2020.</i></p>
        </div>
      </div>
    </div>
  </section>

  <!-- Footer -->
  <footer class="py-5 bg-dark">
    <div class="container">
      <p class="m-0 text-center text-white">Copyright &copy; Tom Hiscock 2020</p>
    </div>
    <!-- /.container -->
  </footer>

  <!-- Bootstrap core JavaScript -->
  <script src="vendor/jquery/jquery.min.js"></script>
  <script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

  <!-- Plugin JavaScript -->
  <script src="vendor/jquery-easing/jquery.easing.min.js"></script>

  <!-- Custom JavaScript for this theme -->
  <script src="js/scrolling-nav.js"></script>

</body>

</html>
